android/src/com/mapswithme/maps/editor/EditorHostFragment.java

```diff
@@ -131,7 +135,9 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
   {
     super.onViewCreated(view, savedInstanceState);
 
-    getToolbarController().getToolbar().findViewById(R.id.save).setOnClickListener(this);
+    mToolbarInnerLayout = getToolbarController().getToolbar().findViewById(R.id.toolbar_inner_layout);
```

Comment:
nit: I would use `final View toolbar = getToolbarController().getToolbar();` to avoid copy-paste.